README.md

```diff
@@ -1,33 +1,74 @@
-# `Turborepo` Vite starter
+# CLASS-web
 
-This is an official starter Turborepo.
+This is an implementation of CLASS that runs entirely in the browser.
 
-## Using this example
+The CLASS web application is available at https://classmodel.github.io/class-web.
 
-Run the following command:
+For more information on CLASS, see https://classmodel.github.io/.
 
-```sh
-npx create-turbo@latest -e with-vite
-```
+## Developers
+
+This repository is a so-called monorepo, where multiple packages and application
+can easily be developed in tandem.
+
+We used [TurboRepo](https://turbo.build/repo) for the initial setup, which uses
+[pnpm workspaces](https://pnpm.io/workspaces) under the hood. As such, it is
+possible to do advanced tricks with Turbo, such as "lint/test/build all
+apps/packages at once with `turbo build`", and share tooling configurations
+across packages/apps, but since this repo is small, we will not rely too much on
+these features.
+
+Currently the repo is home to the following:
 
-## What's inside?
+- packages/
+  - class: reimplementation of CLASS in typescript
+  - config-eslint: shared configuration for eslint
+  - config-typescript: shared configuration for typescript
+- apps/
+  - class-solid: web application with a graphical user interface for CLASS
 
-This Turborepo includes the following packages and apps:
+## Local build
+
+To run a local development version:
+
+```sh
+git clone git@github.com:classmodel/class-web.git
+cd class-web/apps/class-solid
+pnpm install
+pnpm run dev
+```
 
-### Apps and Packages
+## Tech stack
 
-- `docs`: a vanilla [vite](https://vitejs.dev) ts app
-- `web`: another vanilla [vite](https://vitejs.dev) ts app
-- `@repo/ui`: a stub component & utility library shared by both `web` and `docs` applications
-- `@repo/eslint-config`: shared `eslint` configurations
-- `@repo/typescript-config`: `tsconfig.json`s used throughout the monorepo
+The CLASS package is written in typescript. It uses [zod](https://zod.dev/) for
+the configuration and runtime validation. Zod can be converted to/from
+[JSONSchema](https://json-schema.org/), which is ideal for sharing the
+configuration between web-app, library code, and perhaps other implementations
+of CLASS as well.
 
-Each package and app is 100% [TypeScript](https://www.typescriptlang.org/).
+The web application is build with [solid.js](https://docs.solidjs.com/). Solid
+has a relatively simple model for building reactive web applications. With its
```

Comment:
They call them selves a framework at https://docs.solidjs.com/

```suggestion
is a relatively simple framework for building reactive web applications. With its
```

We are already using model for class, let us not used it to describe a framework/library as well.